### PR TITLE
feat(requirements): add system requirements and compatibility page (BCD-259)

### PR DESCRIPTION
**Pull request by tnguyen1**

### CODE REVIEW

File: md/requirements-and-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Requirements and compatibility
+
+This page describes which system requirements need to be met in order to use the Bonita Continuous Delivery Add-On.  
+You will also find information about BCD and Bonita version compatibility.
+
+
+## System requirements
+
+There are different environments where requirements are to be met for BCD:
+- **Control workstation** - the host where your [BCD controller](bcd_controller.md) will run.
```
Comment (JeremJR) [inline md/requirements-and-compatibility.md]: as discussed `Control host` could be more generic

File: md/requirements-and-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+::: warning
+These requirements must be fulfilled when using BCD for **Provisioning**. They apply to target hosts where the Bonita stack will be deployed.
+:::
+
+The system requirements for the target hosts are as follows:
+- Minimum hardware requirements: 1 CPU / 1GiB RAM / 8GiB storage (Equivalent to AWS EC2 `t2.micro` instance)
+- Supported operating systems: Ubuntu-14.04, Ubuntu-16.04
+
+Besides, the following technical requirements must be met:
+- Target hosts must have **Internet access** in order to communicate with Bonitasoft License webservice. It is also required to pull Docker public images for the database instance when applicable.
```
Comment (JeremJR) [inline md/requirements-and-compatibility.md]: Internet access is also used to install packages through apt or pip

**Comment by tnguyen1** [comment]
@JeremJR I have replaced "workstation" with "host" + added that Internet access is required for package installation.

**Review by JeremJR — APPROVED** [review]
thank you

**Pull request merged**